src/CommonProjectProperties.xml

```diff
@@ -0,0 +1,31 @@
+<Project>
+  <!-- Properties common to all projects under this directory -->
+
+  <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
+    <TargetFrameworks>$(TargetCoreFramework);$(TargetDesktopFramework)</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(OS)' != 'Windows_NT'">
+    <TargetFramework>$(TargetCoreFramework)</TargetFramework>
+  </PropertyGroup>
+
+  <!-- Build information -->
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyOriginatorKeyFile>../../Gax.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+  </PropertyGroup>
+
+  <!-- Packaging information -->
+  <PropertyGroup>
+  	<Copyright>Copyright 2017 Google Inc.</Copyright>
+    <Authors>Google Inc.</Authors>
+    <IncludeSymbols>True</IncludeSymbols>
+    <PackageLicenseUrl>https://developers.google.com/open-source/licenses/bsd</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/googleapis/gax-dotnet</PackageProjectUrl>
+    <PackageTags>Google</PackageTags>
```

Comment:
Will this merge with PackageTags in the csproj files, if more tags are wanted for a particular project?
If not, then this should be in the csproj.